lib/membership.js

```diff
@@ -9,20 +9,26 @@ const ensureMembership = async function (personUri, roleUri, organizationUri) {
     PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
     PREFIX dct: <http://purl.org/dc/terms/>
     PREFIX org: <http://www.w3.org/ns/org#>
+    PREFIX adms: <http://www.w3.org/ns/adms#>
 
-    SELECT ?membership ?membershipId
+    SELECT ?membership ?membershipId ?status
     WHERE {
       GRAPH <${USERS_GRAPH}> {
         ?membership a org:Membership ;
           mu:uuid ?membershipId ;
           org:member ${sparqlEscapeUri(personUri)} ;
           org:role ${sparqlEscapeUri(roleUri)} .
         ${organizationStatement}
+        OPTIONAL { ?membership adms:status ?status }
       }
     } LIMIT 1`);
 
   if (queryResult.results.bindings.length) {
     const result = queryResult.results.bindings[0];
+    const status = result.status.value;
+    if (status === ACCESS_BLOCKED_STATUS_URI) {
+      throw new Error(`The user has been blocked and does not have access.`)
```

Comment:
Custom error gooien zodat die in app.js opgevangen kan worden en `403 Forbidden` kan returnen.